Cargo.toml

```diff
@@ -18,6 +18,7 @@ once_cell = "1.21.3"
 clap = { version = "4.5.39", features = ["derive"] }
 
 # Python parsing - using git dependencies as ruff internals are not published to crates.io
+ruff_linter = { git = "https://github.com/astral-sh/ruff/", rev = "0.11.12" }
```

Comment:
📋 **Clarification**: The dependency structure is correct for a Rust workspace. We define `ruff_linter` in the workspace dependencies (here in root Cargo.toml) and reference it with `{ workspace = true }` in the package Cargo.toml. This is the standard pattern that ensures:

1. **Version consistency**: All workspace members use the same version
2. **Single source of truth**: Version is managed in one place  
3. **Reduced duplication**: Avoids repeating git URLs and revisions

The structure follows Rust's workspace dependency best practices. No changes needed.